tests/isSubsetSum_test

```diff
@@ -0,0 +1,16 @@
+import unittest
```

Comment:
Can you write the test case using pytest?
We've moved to automated test builds and use pytest for the same in travis.

Reply:
You can refer this for docs: https://docs.pytest.org/en/latest/